tests/toRdf-manifest.jsonld

```diff
@@ -2251,6 +2251,20 @@
       "purpose": "Verifies that an exception is raised when attempting to redefine @context.",
       "input": "expand/er56-in.jsonld",
       "expectErrorCode": "keyword redefinition"
+    }, {
+      "@id": "#ter57",
+      "@type": ["jld:NegativeEvaluationTest", "jld:ToRDFTest" ],
+      "name": "Array not found for @json @list.",
+      "purpose": "The value for a @json @list term MUST be an array.",
+      "input": "toRdf/er57-in.jsonld",
+      "expectErrorCode": "invalid set or list object"
+    }, {
+      "@id": "#ter58",
+      "@type": ["jld:NegativeEvaluationTest", "jld:ToRDFTest" ],
+      "name": "Array not found for @json @set.",
+      "purpose": "The value for a @json @set term MUST be an array.",
+      "input": "toRdf/er58-in.jsonld",
+      "expectErrorCode": "invalid set or list object"
```

Comment:
```suggestion
```
For the reasons explained above (in expand-manifest.jsonld), I believe that these tests should not be accepted.